packages/js/README.md

```diff
@@ -37,199 +29,155 @@ import { TelnyxRTC } from '@telnyx/webrtc';
+> See [TelnyxRTC.on](./docs/ts/classes/telnyxrtc.md#on) for all events.
 
 ### Calls
 
-To initiate a call from your application:
+To initiate an outgoing call:
 
 ```js
-// You can save this call or wait for `callUpdate` and use the returned `activeCall`
 const call = client.newCall({
   // Destination is required and can be a phone number or SIP URI
   destinationNumber: '18004377950',
-  callerName: 'Caller ID Name',
-  // Caller ID number is optional.
-  // You can only specify a phone number that you own and have assigned
-  // to your Connection in the Telnyx Portal
-  callerNumber: '‬',
-  audio: true,
-  video: false, //Used to enable/disable video
+  callerNumber: '‬155531234567',
 });
 ```
 
-A [Call](./docs/ts/interfaces/icall.md) has methods that can be hooked up to your UI:
+To enable video when calling:
 
 ```js
-// End a call
-call.hangup();
-
-// Call states that can be toggled
-call.hold();
-call.unhold();
-call.muteAudio();
-call.unmuteAudio();
-
-// Send digits and keypresses
-call.dtmf('1234');
+const videoCall = client.newCall({
+  destinationNumber: 'sip:example@sip.example.com',
+  video: true,
+});
 
-// Answer an incoming call
-call.answer();
-// Hangup (reject) an incoming call
-call.hangup();
+// And in your HTML, replace the audio element with video.
+//  <video id="remoteMedia" autoplay="true" playsinline="true" />
 ```
 
----
+> See [TelnyxRTC.newCall](./docs/ts/classes/telnyxrtc.md#newCall) for all options.
 
-#### Using WebRTC with React to make an audio call
+To answer an incoming call:
 
-```Js
-    // Used to set remote stream
-    if (mediaRef.current && call && call.remoteStream) {
-        mediaRef.current.srcObject = call.remoteStream;
-    }
+```js
+client.on('telnyx.notification', (notification) => {
+  const call = notification.call;
 
-    <audio
-      ref={mediaRef}
-      id='audioCall'
-      autoPlay='autoplay'
-      controls={false}
-    />
+  if (notification.type === 'callUpdate' && call.state === 'ringing') {
+    call.answer();
+  }
+});
 ```
 
----
+Both the outgoing and incoming `Call` instance has methods that can be hooked up to your UI:
 
-#### Using WebRTC with Vanilla Javascript to make video call
-
-```Js
-    // Used to set local and remote stream
-    client.remoteElement = 'remoteVideo';
-    client.localElement = 'localVideo';
+```js
+// Hangup or reject an incoming call
+call.hangup();
 
-    <video
-      id="localVideo"
-      autoplay="true"
-      playsinline="true"
-    />
-     <video
-      id="remoteVideo"
-      autoplay="true"
-      playsinline="true"
-    />
+// Send digits and keypresses
+call.dtmf('1234');
 
+// Call states that can be toggled
+call.hold();
+call.muteAudio();
 ```
 
```

Comment:
What do yu think to keep the main methods here, for example:

```
call.answer()
call.newCall()
....
```

Reply:
`newCall` and `answer` were just used at the top. I think it is fine removing them from this last example.